src/MartinGeorgiev/Doctrine/ORM/Query/AST/Functions/BaseFunction.php

```diff
@@ -43,12 +44,14 @@ protected function addNodeMapping(string $parserMethod): void
 
     public function parse(Parser $parser): void
     {
+        $ormV2 = !\class_exists(TokenType::class);
```

Comment:
As this expression repeats at several places, wdyt about guaranteeing the expression will always remain consistent by extracting into a new [Utils](https://github.com/martin-georgiev/postgresql-for-doctrine/tree/main/src/MartinGeorgiev/Utils) helper class (that will go away one day)?
```php
namespace MartinGeorgiev\Utils;

class DoctrineOrmVersion {
   public static function isVersion3(): bool {
      return \class_exists(\Doctrine\ORM\Query\TokenType::class);
   }
}
```

Reply:
As Doctrine has now added `TokenType` in the unreleased `2.19.x` branch, I've gone for `isPre219()`, but welcome better naming ideas!